.github/workflows/checker-framework.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Execute checker framework
-        run: groovy ./.ci/checker-framework.groovy ${{ matrix.profile }}
+        run: groovy ./.ci/checker.groovy ${{ matrix.profile }}
         env:
+      - name: analyse
           MAVEN_OPTS: "-Xmx4g"
```

Comment:
This is not a place where we need it. This name is shown in log page, there are plenty of space

Reply:
So on which file the following code should be put in?